### ISSUE TITLE
Does this work?

### ISSUE THREAD

**Issue by LordRampantHump** [pixijs/pixi-tilemap#50]
I dont see the map render at all here: http://pixijs.io/pixi-tilemap/?canvas

Am I missing something?

**Comment by LordRampantHump** [comment]
commenting this line in main.js allows the map to render : _renderer.gl.flush();

**Comment by ivanpopelyshev** [comment]
yep, you are right, need to fix it. there's no "gl" in canvas mode.

**Comment by ivanpopelyshev** [comment]
Also, here's tutorial: https://github.com/Alan01252/pixi-tilemap-tutorial
